Support for AlmaLinux / Rocky Linux

A couple of our vendors have based their AMI's on AlmaLinux or Rocky Linux. They should be binary compatible so shouldn't need additional testing.

@mattgresham I've been working on this, and started to test, but looks like almalinux & rockly linux are[ not supported by distributor](https://docs.aws.amazon.com/systems-manager/latest/userguide/distributor.html#:~:text=platforms%20and%20architectures.-,Supported%20package%20platforms%20and%20architectures,-You%20can%20use). 

```
[ERROR] Failed to build and publish new distributor package version in region us-east-1. Error An error occurred (InvalidDocumentContent) when calling the CreateDocument operation: Unsupported Platform included in manifest: almalinux.
```

--- a/official-package/README.md
+++ b/official-package/README.md
@@ -18,17 +18,18 @@ If you have a question checkout the [FAQ](#faq) to see if it has already been an
 | -------------------------------------------------------- | ------------- |
 | Amazon Linux 2                                           | x86_64, arm64 |
 | Amazon Linux 2023                                        | x86_64, arm64 |
-| Red Hat Enterprise Linux 6                               | x86_64        |
 | Red Hat Enterprise Linux 7                               | x86_64        |
 | Red Hat Enterprise Linux 8                               | x86_64, arm64 |
 | Red Hat Enterprise Linux 9                               | x86_64        |
-| CentOS 6                                                 | x86_64        |
+| Rocky Linux 8                                            | x86_64, arm64 |
+| Rocky Linux 9                                            | x86_64, arm64 |
+| AlmaLinux 8                                              | x86_64, arm64 |
+| AlmaLinux 9                                              | x86_64, arm64 |
 | CentOS 7                                                 | x86_64        |
 | CentOS 8                                                 | x86_64, arm64 |
 | Oracle Enterprise Linux 6                                | x86_64        |
 | Oracle Enterprise Linux 7                                | x86_64        |
 | Oracle Enterprise Linux 8                                | x86_64        |
-| SUSE Linux Enterprise Server 11                          | x86_64        |
 | SUSE Linux Enterprise Server 12                          | x86_64        |
 | SUSE Linux Enterprise Server 15                          | x86_64        |
 | Ubuntu 16.04                                             | x86_64        |

